File: algoritmo_e_logica/python/q_6.py
# Crie uma função que receba um nome como parâmetro e retorne uma mensagem personalizada no seguinte formato: "Olá, (nome), que bom ter você no programa Trilhas."

def capitalizar_nome(nome):
    return nome.capitalize()

def mensagem_personalizada(nome):
    nome_formatado = capitalizar_nome(nome)
    return f'✨ Olá, {nome_formatado}! Que bom ter você no programa Trilhas. 🚀'

nome = input('Digite seu nome: ').strip()
print(mensagem_personalizada(nome))

File: algoritmo_e_logica/javascript/q_6.js
// Crie uma função que receba um nome como parâmetro e retorne uma mensagem personalizada no seguinte formato: "Olá, (nome), que bom ter você no programa Trilhas."

const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

function capitalizarNome(nome) {
    return nome.charAt(0).toUpperCase() + nome.slice(1).toLowerCase();
}

function mensagemPersonalizada(nome) {
    let nomeFormatado = capitalizarNome(nome);
    return `✨ Olá, ${nomeFormatado}! Que bom ter você no programa Trilhas. 🚀`;
}

rl.question('Digite seu nome: ', (nome) => {
    console.log(mensagemPersonalizada(nome));
    rl.close();
});